js/views/userPage/UserPage.js

```diff
@@ -47,6 +61,18 @@ export default class extends baseVw {
     if (this.curConn && this.curConn.server) {
       this.showStoreWelcomeCallout = !this.curConn.server.get('dismissedStoreWelcome');
     }
+
+    this.listenTo(app.ownFollowing, 'add', this.onOwnFollowingAdd);
+    this.listenTo(app.ownFollowing, 'remove', this.onOwnFollowingRemove);
+
+    this.followsYou = false;
+    followsYou(this.model.id).done(data => {
+      if (this.miniProfile) {
+        this.miniProfile.setState({ followsYou: data.followsMe });
+      }
+
+      if (this.followerCount === 0) this.followerCount += 1;
```

Comment:
I'll fix that. It should be if your on someone else's user page and they follow you, but their cached following count is 0, it should be 1 (because we know they're following at least one person).